features/wasm-typed-fun-refs.yml

```
@@ -0,0 +1,6 @@
+name: Typed function references (WebAssembly)
+description: Function references that are typed and that can be called directly.
```

We usually have whole sentences for these. I tried my best. 😅 

```suggestion
description: A typed function reference can be called directly.
```